File: lib_common/src/main/java/com/wujia/lib_common/utils/StringUtil.java
package com.wujia.lib_common.utils;

import java.util.Locale;

/**
 * author ：shenbingkai@163.com
 * date ：2019-01-27
 * description ：
 */
public class StringUtil {


    public static String format(String format, Object... args) {
        return String.format(Locale.CHINA, format, args);
    }
}

File: lib_common/src/main/java/com/wujia/lib_common/utils/StringUtil.kt
package com.wujia.lib_common.utils

import java.util.Locale

/**
 * author ：shenbingkai@163.com
 * date ：2019-01-27
 * description ：
 */
object StringUtil {


    fun format(format: String, vararg args: Any): String {
        return String.format(Locale.CHINA, format, *args)
    }
}
